price-feeder/oracle/provider/websocket_controller.go

```diff
@@ -190,3 +240,14 @@ func (wsc *WebsocketController) reconnect() {
 	go wsc.Start()
 	telemetryWebsocketReconnect(wsc.providerName)
 }
+
+func (wsc *WebsocketController) pongHandler(appData string) error {
+	return nil
+}
```

Comment:
Do we need this function?

Reply:
No, we don't. It's just nice to have there for when you need to test a provider and see if they pong you back. I think it can be removed.

Reply:
https://github.com/umee-network/umee/pull/1554/commits/166014a0f416f06f581659b8dea14a521c7e3d42